deploy/crs/prometheus/prometheus_rules/sre/check-full-cust-pv.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-filesysfull
+    role: alert-rules
+  name: sre-filesysfull
+spec:
+  groups:
+  - name: sre-filesysfull
+    rules:
+      # This matches with the default KubePersistentVolumeUsageCritical; this rule inverts the regex match.
+    - alert: KubeCustomerPersistentVolumeUsageCritical
```

Comment:
The group has a prefix of `sre-` but wonder if we need to also prefix alerts with something like `Sre`.  This goes back to the question of how we'll ensure PE rules and alerts don't conflict / overlap with SRE rules and alerts.

I have deployed this file twice, one with the `sre` prefix and one with `sre2`.  I haven't been able to review alerts from a UI point of view (need fix for auth bug to land) but in-cluster it appears to be happy...

Reply:
You "should" (scare quotes) be able to use the Alertmanager API to query the status of those things? I'm not sure, and that might be harder than the other auth problem.

On the rule name, I do wonder what the chances of two groups creating a rule with the same name are. The more I think about it the more I like the idea of heading it off at the pass and having our team name in there somewhere.

After lunch today I'll put up a change.